TestsFoundation/TestsFoundation/Element/Element.swift

```diff
@@ -197,6 +197,28 @@ public extension Element {
         return false
     }
 
+    @discardableResult
+    func swipeUntilVisible(direction: SwipeDirection = .up, timeout: TimeInterval = 15, gracePeriod: UInt32 = 1) -> Bool {
+        let deadline = Date().addingTimeInterval(timeout)
+        while Date() < deadline {
+            if self.isVisible {
+                return true
+            }
+            switch direction {
+            case .down:
+                app.swipeDown()
+            case .up:
+                app.swipeUp()
+            case .left:
+                app.swipeLeft()
+            case .right:
+                app.swipeRight()
```

Comment:
As I remember this will swipe at the center of the app so if we are in a split view on an iPad it most likely will scroll in the details area. A more future proof solution would be to scroll on the container of the element. It works for now just keep this in mind for the future.